install.sh

```diff
@@ -150,8 +150,8 @@ gum_no_tty() {
 get_gum() {
 	if test ! -t 1 -o "$GUM" = "0"; then
 		GUM=gum_no_tty
-	elif which gum >/dev/null 2>&1; then
-		GUM=$(which gum)
+	elif command -v gum >/dev/null 2>&1; then
+		GUM=$(command -v gum)
```

Comment:
```suggestion
		GUM=gum
```

If it exists we just use it, and it's the user's problem if they have `gum` aliased to something broken.

Reply:
documentation for `command` was surprisingly harder to come by than expected. if needs be, we can do some processing on `-V`, but just passing `gum` through should be fine until we find otherwise.

Reply:
Yeah, you can do `man zshbuiltins` or `man bash` (and then search for BUILTIN COMMANDS). It is tricky to understand but it's portable between at least these two shells. Not sure about `fish`.

Reply:
Well, we run it in `sh` and anyone who has `fish` aliased to `sh` is just taking their life in their hands.